core/src/io.jl

```diff
@@ -181,29 +181,37 @@ function flow_table(
 }
     (; config, saved, integrator) = model
     (; t, saveval) = saved.flow
-    (; connectivity) = integrator.p
-    (; graph) = connectivity
+    (; graph) = integrator.p
+    (; flow_dict, flow_vertical_dict) = graph[]
 
-    I, J, _ = findnz(get_tmp(connectivity.flow, 0))
     # self-loops have no edge ID
-    # unique_edge_ids = [get(connectivity.edge_ids_flow, ij, missing) for ij in zip(I, J)]
-    unique_edge_ids_flow = Union{Int, Missing}[]
-    for (i, j) in zip(I, J)
-        if i == j
-            push!(unique_edge_ids_flow, missing)
-        else
-            edge_metadata = metadata_from_edge(graph, Edge(i, j))
-            push!(unique_edge_ids_flow, edge_metadata.id)
-        end
+    from_node_id = Int[]
+    to_node_id = Int[]
+    unique_edge_ids_flow = Vector{Union{Int, Missing}}()
+    flow_vertical_dict_inverse = Dict(value => key for (key, value) in flow_vertical_dict)
+    flow_dict_inverse = Dict(value => key for (key, value) in flow_dict)
+
+    for i in 1:length(flow_vertical_dict_inverse)
+        id = flow_vertical_dict_inverse[i]
+        push!(from_node_id, id.value)
+        push!(to_node_id, id.value)
+        push!(unique_edge_ids_flow, missing)
     end
 
-    nflow = length(I)
+    for i in 1:length(flow_dict_inverse)
+        from_id, to_id = flow_dict_inverse[i]
+        push!(from_node_id, from_id.value)
+        push!(to_node_id, to_id.value)
+        push!(unique_edge_ids_flow, graph[from_id, to_id].id)
+    end
```

Comment:
```suggestion
    for (from_id, to_id) in flow_dict_inverse
        push!(from_node_id, from_id.value)
        push!(to_node_id, to_id.value)
        push!(unique_edge_ids_flow, graph[from_id, to_id].id)
    end
```

Reply:
So you verified that iterating through it and indexing one-by-one doesn't behave the same?